styx-scheduler-service/src/main/java/com/spotify/styx/state/PersistentStateManager.java

```diff
@@ -146,6 +146,8 @@ private void tickInstance(RunState state) {
       outputHandler.transitionInto(state);
     } catch (StateTransitionConflictException e) {
       log.debug("State transition conflict when ticking instance: {}", state.workflowInstance(), e);
+    } catch (CounterCapacityException e) {
```

Comment:
Might it be more suitable to handle `CounterCapacityException` in `KubernetesDockerRunner.emitPodEvents` ?